lib/python/rose/config_editor/valuewidget/format.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, value, metadata, set_value, hook, arg_str=None):
         self.entry_table = gtk.Table(rows=num_entries + 1, columns=1)
         self.entry_table.show()
         self.entries = []
-        for r, format_name in enumerate(value.split()):
+        for format_name in value.split():
```

Comment:
I can also see a lot of unnecessary lambda functions below this, on lines 64-69, which you might want to remove.